config/default_records/submission_templates/014_lcm_triomics_submission_templates.wip.yml

```diff
@@ -0,0 +1,9 @@
+# Submission templates for LCM Triomics WGS and EMSeq
+---
+# LCM Triomics EMSeq submission template
+Limber-Htp - LCM Triomics EMSeq:
+  submission_class_name: "LinearSubmission"
+  related_records:
+    request_type_keys: ["limber_lcm_triomics_emseq"]
+    product_line_name: LCM Triomics WGS and EMSeq
```

Comment:
I would rename these to just LCM Triomics too

Reply:
I have renamed them both catalogue and product line to just LCM Triomics now.